docs/source/fundamentals/platform.md

```diff
@@ -0,0 +1,42 @@
+---
+title: The Apollo platform
+description: How Apollo helps you run GraphQL in production with confidence
```

Comment:
I think this description should be made a bit more general, since Apollo as a whole isn't just about production - it also helps you get started